### PR TITLE
Update the project to use the Kirkstone release of Yocto

### LINKED ISSUE #34: update yocto release to kirkstone
Follow up for #31

### PR DESCRIPTION
**Pull request by khronozz**
This change resolves #34 and #24 by making the necessary changes for the Kirkstone update and migrating the services used in the challenges from sysvinit to systemd.

### CODE REVIEW

**Comment by nimarty** [comment]
Checks fail because disk space limit of github actions workspace is reached.

**Comment by khronozz** [comment]
One solution to this would be to self-host a GitHub runner.
https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners

File: manifest.xml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+
   <!-- Yocto -->
-  <remote name="yoctoproject" 
+  <remote name="yoctoproject"
           fetch="git://git.yoctoproject.org/" />
   <project name="poky"
-           remote="yoctoproject" 
-           revision="7f9b7f912e13451a4cd03b10a8090a5def68dc39"
+           remote="yoctoproject"
+           revision="84dd3d0e6c90656f2a7105aabd303a62fa49eeba"
```
[the remark this answers is not in the document]
Comment (khronozz) [reply manifest.xml:9]: Right, I noticed that the other repositories had no tag available, but it makes sense for this one since a tag is available.

**Comment by nimarty** [comment]
The build is passing on a self-hosted runner, but now there is a new issue with pushing the hacky-packages-server docker image:
`Error: buildx failed with: ERROR: denied: installation not allowed to Write organization package`. **Note**: The hacky-packages-server build job is still running on github runner, not self-hosted.

**Pull request merged**